Assets/MRTK/Core/Providers/Hands/BaseHand.cs

```diff
@@ -11,7 +11,30 @@ public abstract class BaseHand : BaseController, IMixedRealityHand
         // Hand ray
         protected virtual IHandRay HandRay { get; } = new HandRay();
 
-        public override bool IsInPointingPose => HandRay.ShouldShowRay;
+        public override bool IsInPointingPose
+        {
+            get
+            {
+                if(!HandRay.ShouldShowRay) return false;
+
+                if (!TryGetJoint(TrackedHandJoint.Palm, out var palmPose)) return false;
+
+                Camera mainCamera = CameraCache.Main;
+
+                if (mainCamera == null)
+                {
+                    return false;
+                }
+
+                Transform cameraTransform = mainCamera.transform;
+
+                Vector3 projectedPalmUp = Vector3.ProjectOnPlane(-palmPose.Up, cameraTransform.up);
+
+                // We check if the palm forward is roughly in line with the camera lookAt
+                // We must also ensure we're not in teleport pose
+                return Vector3.Dot(cameraTransform.forward, projectedPalmUp) > 0.3f;
```

Comment:
Spotted a magic number. Thoughts on making this configurable, or at least moved into a constant or variable? Consider representing it as an angle rather than dot product, to make it more human-understandable.

Reply:
It also might be more clear to say something like 

`return HandRay.ShouldShowRay && !Hand.IsInTeleportPose`

Assuming that a property `IsInTeleportPose` exists :-)

Reply:
Yeah this is a somewhat lingering effect of when I tried moving the teleport pose into the basehand rather than our current design pattern of having everything handled by the Articulated Hand definition, I'll try refactoring this one out into that class as well to keep things consistent across all hand implementations.

Reply:
Ultimately just reverting this change to BaseHand so as to not affect the existing behavior. The classes that want to take advantage of the ArticulatedHandDefintion's InPointingPose already do so.